lightapi/core.py

```
@@ -336,3 +467,134 @@ def process(self, request, response):
             The response (possibly modified) or None to continue processing.
         """
         return response
+
+
+class CORSMiddleware(Middleware):
```

Ruff (0.11.9)

Redefinition of unused `CORSMiddleware` from line 8

(F811)